tools/FindPythonLibsNew.cmake

```diff
@@ -147,7 +147,7 @@ string(REGEX REPLACE "\\\\" "/" PYTHON_PREFIX "${PYTHON_PREFIX}")
 string(REGEX REPLACE "\\\\" "/" PYTHON_INCLUDE_DIR "${PYTHON_INCLUDE_DIR}")
 string(REGEX REPLACE "\\\\" "/" PYTHON_SITE_PACKAGES "${PYTHON_SITE_PACKAGES}")
 
-if(CMAKE_HOST_WIN32 AND NOT (MINGW AND DEFINED ENV{MSYSTEM}))
```

Comment:
The `MINGW` part of the condition here disappeared, now, though, and doesn't come back later. Why is that?

Reply:
If I understand correctly, this `MINGW` condition was here so that when using windows+msys+mingw it would *only* look for msys system python, while when using windows+msys+any other compiler, (e.g. MSVC) it would *only* look for windows python (<https://github.com/pybind/pybind11/pull/2087>)

Now the logic on windows is to first look for the windows python lib, and if that fails and we are using msys, then look for the msys system python lib. So the `MINGW` condition is gone - the logic is the same regardless of which compiler is being used with msys.

Reply:
OK, yes. And - please bear with me; I'm really not experienced with msys - does it make sense that `MSVC` would go looking for msys system Python, then (if no Windows Python is found)?

Reply:
That's a good point - I don't think that would make sense. Then it would be better to put the `MINGW` condition back in.

Reply:
Alright, great, thanks! Final thing: which of the two should get precedence when using mingw, the Windows Python or the msys Python?

Reply:
I think this will be determined by which python binary comes first in the users `PATH` (or whatever they set explicitly with `-DPYTHON_EXECUTABLE=...`)

Reply:
OK, then, then I'm in! :-)